06week/fizzbuzz.js

```diff
@@ -0,0 +1,47 @@
+//solution 1: while statement with if/else
+var num = 1;
+
+while (num<=1000){
+  if (num%3===0 && num%5===0) {
+    console.log("FizzBuzz");
+  } else if (num%3===0) {
+    console.log("Fizz");
+  } else if (num%5===0) {
+    console.log("Buzz");
+  } else {
+    console.log(num);
+  };
+  ++num;
+};
+
+
+//solution 2: for statement with if/else
+for (i=1; i<=1000; i++){
+  if (i%3===0 && i%5===0) {
+    console.log("FizzBuzz");
+  } else if (i%3===0) {
+    console.log("Fizz");
+  } else if (i%5===0) {
+    console.log("Buzz");
+  } else {
+    console.log(i);
+  };
+};
+
+
+//solution 3: for statement with independent if statements
+var print = ""
```

Comment:
nice, but i would instantiate ```var print=""``` at the beginning of the for loop.
and console log it at the end of the for loop. 
Instead of before and after the for loop.